fars/booking/migrations/0015_auto_20230201_2216.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.16 on 2023-02-01 20:16
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
It would be good to drop all migrations you've created during development and add only one new migration for the whole feature.